Disable Security Mitigations/disablesecurity.txt

```diff
@@ -1,11 +1,11 @@
-Disabling Spectre, Meltdown and Zombieload MDS mitigations may improve your CPU speed But these are security risks for certain workloads, and disabling the mitigations is purely all at your own risk=
-Mitigations currently disabled=
-Mitigations currently enabled=
-Status=
-will be updated after rebooting the server=
-Enable Mitigations=
-Disable Mitigations=
-Disable Security Mitigations A Reboot Is Required For Changes To Take Effect=
-A Reboot Is Required For Changes To Take Effect=
-Mitigations will be enabled after a reboot=
-Mitigations will be disabled after a reboot=
+Disabling Spectre, Meltdown and Zombieload MDS mitigations may improve your CPU speed But these are security risks for certain workloads, and disabling the mitigations is purely all at your own risk=Deshabilitar las mitigaciones de Spectre, Meltdown y Zombieload MDS puede mejorar la velocidad del CPU, estos son riesgos de seguridad para ciertas cargas de trabajo; La desactivación de las mitigaciones es bajo su propio riesgo
```

Comment:
Would make the following changes: `Deshabilitar las mitigaciones de Spectre, Meltdown y Zombieload MDS puede mejorar la velocidad del CPU, pero hay riesgos de seguridad para ciertas cargas de trabajo; La desactivación de las mitigaciones es bajo su propio riesgo`

Reply:
Got it!